Lombiq.HelpfulExtensions.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -0,0 +1,95 @@
+using Atata;
+using Lombiq.HelpfulExtensions.Tests.UI.Constants;
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using Shouldly;
+using System.Threading.Tasks;
+using static Lombiq.HelpfulExtensions.Tests.UI.Constants.TextInputValues;
+using static Lombiq.HelpfulExtensions.Tests.UI.Constants.XPathSelectors;
+
+namespace Lombiq.HelpfulExtensions.Tests.UI.Extensions;
+
+public static class TestCaseUITestContextExtensions
+{
+    public static async Task TestFeatureWidgetsAsync(this UITestContext context)
+    {
+        await context.SignInDirectlyAsync();
+        await context.EnableFeatureDirectlyAsync(FeatureIds.ContentTypes);
+        await context.EnableFeatureDirectlyAsync(FeatureIds.Widgets);
+
+        var widgets = new[]
+        {
+            WidgetTypes.Container,
+            WidgetTypes.Html,
+            WidgetTypes.Liquid,
+            WidgetTypes.Markdown,
+            WidgetTypes.Menu,
+        };
+
+        foreach (var widget in widgets)
+        {
+            await context.TestWidgetAsync(widget);
+        }
+    }
+
+    public static async Task TestFlowAdditionalStylingPartNotActivatingGh76Async(this UITestContext context)
```

Comment:
"Gh76"?

Reply:
The GitHub issue code. Some open-source projects use issue code in test names, while others don't. I didn't find any suggestions about this. I can remove it.

Reply:
Referencing the issue is a good idea, but this way is not obvious. I'd prefer if you added a `<summary>` above this method that included the issue title and its URL.